vnext/ReactUWP/Modules/NetworkingModule.cpp

```diff
@@ -263,7 +263,7 @@ void NetworkingModule::NetworkingHelper::SendRequest(const std::string& method,
         else if (_stricmp(name.c_str(), "content-encoding") == 0)
           contentEncoding = value;
         else
-          request.Headers().Append(facebook::react::unicode::utf8ToUtf16(name), facebook::react::unicode::utf8ToUtf16(value));
+          request.Headers().TryAppendWithoutValidation(facebook::react::unicode::utf8ToUtf16(name), facebook::react::unicode::utf8ToUtf16(value));
```

Comment:
>TryAppendWithoutValidation [](start = 28, length = 26)

Would it be safer to only use TryAppendWithoutValidation() if name == "Authorization", otherwise fallback to use Append()?

Reply:
If you feel that is the right call I am fine with making that change.

I personally think that the validation performed by Append is unexpected.
There are a couple more headers that can cause this as well, HttpHeaderContentCollection expects consumers to use the properties for any value that has a property.
https://docs.microsoft.com/en-us/uwp/api/windows.web.http.headers.httpcontentheadercollection.contentdisposition

I believe that using TryAppendWithoutValidation gives the most control to the developer and will prevent this from coming up again.

But I acknowledge that this is not my decision to make :) and if you feel strongly that this should be scoped to Authorization headers I am happy to make that change.

Reply:
My main concern is security - if the validation done by Append is adding protection from security vulnerabilities then I'd prefer to use it as much as possible.  I honestly don't know what validation it's performing though.  If there's no real security aspect then your change is fine as is.

Reply:
That's a fair point.
I tried to skim through the source for Append vs TryAppendWithoutValidation, but there is too much abstraction to easily be able to say that this is generically safe.
I don't believe there is a security aspect, but I'm not a security expert.

I think you are correct, the right thing to do in this case is to only use TryAppendWithoutValidation for the Authentication header. I'll update the PR today.
Thanks for taking a look!

Reply:
Apologies for the delay on getting the update. But better than never (I hope :) )

Reply:
Thanks!  good to go

Reply:
awesome thanks!